driver/normalizer/normalizer.go

```diff
@@ -570,8 +586,10 @@ var Normalizers = []Mapping{
 		Obj{
 			"Statements": Var("stmts"),
 			// TODO(dennwc): remap to custom positional fields
-			"OpenBraceToken":  Any(),
-			"CloseBraceToken": Any(),
+			"OpenBraceToken":     Any(),
```

Comment:
Probably is ok to just drop them - those are syntactic tokens, not semantic elements and the nodes already have positions.

Reply:
Totally agree on this one, this is why it's only a "soft" TODO instead of FIXME.

It may be nice though to include those token positions in the `@pos` map, similar to what Go driver does. But this one is for later when I'll have nothing else to do :)

Reply:
Opened an issue in the guide to clarify the convention: https://github.com/src-d/guide/issues/327